packages/app/src/app/store/sequences.js

```diff
@@ -107,7 +107,8 @@ export const forkSandbox = sequence('forkSandbox', [
   set(state`editor.isForkingSandbox`, true),
   actions.forkSandbox,
   actions.moveModuleContent,
-  set(state`editor.sandboxes.${props`sandbox.id`}`, props`sandbox`),
+  set(props`sandbox`, props`forkedSandbox`),
```

Comment:
Because with plain mobx we have to explicitly convert `npmDependencies` to a Dictionary, because it arrives as a plain object. With mobx-state-tree this seems to happen automatically with the "typing" model. But now with plain mobx we have to be explicit about this. Check `addSandbox` action... that is where it sets `npmdependencies` to dictionary